content/_data/changelogs/lts.yml

```diff
@@ -2837,6 +2837,170 @@
         These were made to anticipate a bugfix in Firefox which has been backed out since.
         (regression in 2.164.3)
 
+- version: 2.204.1
+  date: 2019-12-18
+  banner: >
+    <b>Configuration as code plugin</b> users upgrading to Jenkins 2.204.1 should review the
+    <ul>
+    <li><a href="/doc/upgrade-guide/2.204/#global-configuration-save-and-configuration-as-code-plugin">LTS upgrade guide global configuration save</a> topic</li>
+    <li><a href="/doc/upgrade-guide/2.204/#remove-browser-based-metadata-download-settings">LTS upgrade guide download settings</a> topic</li>
+    </ul>
```

Comment:
This banner takes up a _lot_ of space for a plugin installed on 3% of instances. Do we really want to create the expectation that this is the level of service users can expect?

Reply:
Fixed in 1301f3665